benefits/static/css/styles.css

```diff
@@ -115,6 +113,7 @@ footer .footer-links a {
 
 .btn-link {
   color: #046b99;
+  font-weight: bold;
```

Comment:
This declaration changes the Eligibility Start page

<img width="1303" alt="image" src="https://user-images.githubusercontent.com/3673236/164304501-f17942ae-1070-49f4-8d46-bb406116c272.png">

So maybe the `font-weight: bold` should style should be declared somewhere where it only touches the Help page links